lib/content_encoding.c

```diff
@@ -270,10 +308,11 @@ static CURLcode gzip_init_writer(struct connectdata *conn,
     zp->zlib_init = ZLIB_INIT_GZIP; /* Transparent gzip decompress state */
   }
   else {
-    /* we must parse the gzip header ourselves */
+    /* we must parse the gzip header and trailer ourselves */
     if(inflateInit2(z, -MAX_WBITS) != Z_OK) {
       return process_zlib_error(conn, z);
     }
+    zp->trailerlen = 8;          /* Trailer length. */
```

Comment:
where did you get this magic number

Reply:
http://www.zlib.org/rfc-gzip.html#file-format

A trailer is made of a CRC32 and a 32-bit input size.

Reply:
I see. I had actually looked at that before I asked and couldn't figure it out. I realize now the `(more -->)` is describing the structure and so trailer is after `...compressed blocks...`.